### PR TITLE
Rename openFileInfo to fileInfo

### PR DESCRIPTION
**Pull request by artonge**
And check if openFile is set in the URL before opening a file.

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
@ShGKme because we talked about this iirc

File: apps/files/src/components/FilesListVirtual.vue [hunk 1]
```diff
@@ -207,7 +207,7 @@ export default defineComponent({
 		mainContent.addEventListener('dragover', this.onDragOver)
 
 		// handle initially opening a given file
-		const { id } = loadState<{ id?: number }>('files', 'openFileInfo', {})
+		const { id } = loadState<{ id?: number }>('files', 'fileInfo', {})
```
Comment (ShGKme) [inline apps/files/src/components/FilesListVirtual.vue:210]: Comment is not actual anymore 👀

**Review by ShGKme — APPROVED** [review]
Thanks 💙

In which case do we have no `fileid` from route and initial state is required? Except the public sharing.

**Comment by artonge** [comment]
> In which case do we have no `fileid` from route and initial state is required? Except the public sharing.

I don't remember, but I think this use case does not exists anymore with F2V.

**Pull request merged**